python/ray/reporter.py

```diff
@@ -107,6 +112,24 @@ def __init__(self, redis_address, redis_password=None):
     def get_cpu_percent():
         return psutil.cpu_percent()
 
+    @staticmethod
+    def get_gpu_usage():
+        if gpustat is not None:
+            gpu_utilizations = []
+            gpus = []
+            try:
+                gpus = gpustat.new_query().gpus
+            except Exception:
+                logger.debug("GPUtil failed to retrieve GPUs.")
+            for gpu in gpus:
+                # Note the keys in this dict have periods which throws
+                # off javascript so we change .s to _s
+                gpu_data: Dict[str, Any] = {'_'.join(key.split('.')): val 
+                    for key, val in gpu.entry.items()}
+                gpu_utilizations.append(gpu_data)
+            return gpu_utilizations
+        raise RuntimeError('Where is your GPUUtil young man')
```

Comment:
I think you forgot to change this message? lol

Reply:
Hahaha, accurate.